python/tvm/auto_scheduler/compute_dag.py

```diff
@@ -32,7 +32,12 @@
 
 
 class LayoutRewriteOption:
-    """Options for applying layout rewrite."""
+    """
+    Options for applying layout rewrite.
+
+    The NO_REWRITE and INSERT_TRANSFORM_STAGE is expected to be used when tuning a dependent op,
+    and the REWRITE_FOR_PRE_TRANSFORMED is expected to be used when tuning ops inside a network.
```

Comment:
"dependent op" may be confusing. Could you change to another term that against to "ops inside a network"?

```suggestion
    The NO_REWRITE and INSERT_TRANSFORM_STAGE are expected to be used when tuning a dependent op,
    and the REWRITE_FOR_PRE_TRANSFORMED is expected to be used when tuning ops inside a network.
```

Reply:
"a dependent op" -> "an independent op"

Reply:
Ah that makes sense...maybe "a standalone op" would be more straightforward?